search-app/src/cmr/search/routes.clj

```diff
@@ -62,6 +62,31 @@
                 :body-copy body
                 :body (java.io.ByteArrayInputStream. (.getBytes body)))))))
 
+
+(defn add-equal-sign-to-params-in-list
+  "Add = sign to each parameter in the param-list if it doesn't already exist."
+  [param-list]
+  (map #(if (and (string? %)
+                 (not= "" (string/trim %))
+                 (not (string/includes? % "=")))
+          (str % "=")
+          %) param-list)) 
```

Comment:
param-list should be on a line by itself

Reply:
Done.